tools/targets/lint.py

```diff
@@ -0,0 +1,262 @@
+"""A linting utility for targets.json
+
+This linting utility may be called as follows:
+python <path-to>/lint.py TARGET [TARGET ...]
+
+all targets will be linted
+"""
+
+from os.path import join, abspath, dirname
+if __name__ == "__main__":
+    import sys
+    ROOT = abspath(join(dirname(__file__), "..", ".."))
+    sys.path.insert(0, ROOT)
+from copy import copy
+from yaml import dump_all
+import argparse
+from tools.targets import Target, set_targets_json_location, TARGET_MAP
```

Comment:
We do system then local (pylint prefers this style by default, and changing pylint's defaults seems bad). I'll add a newline.